src/behavior/src/A32NX_Interior_Misc.xml

```diff
@@ -301,21 +301,21 @@
     Template for console light switches
     -->
     <Template Name="FBW_LIGHTING_Switch_Console_Template">
-        <UseTemplate Name="ASOBO_LIGHTING_Switch_Light_Template">
-            <DIMMER/>
-            <ANIMTIP_0_ON_PERCENT>0</ANIMTIP_0_ON_PERCENT>
-            <ANIMTIP_1_ON_PERCENT>0.5</ANIMTIP_1_ON_PERCENT>
-            <ANIMTIP_2_ON_PERCENT>1</ANIMTIP_2_ON_PERCENT>
-            <ANIMTIP_0>Set CONSOLE/FLOOR OFF</ANIMTIP_0>
-            <ANIMTIP_1>Set CONSOLE/FLOOR DIM</ANIMTIP_1>
-            <ANIMTIP_2>Set CONSOLE/FLOOR BRT</ANIMTIP_2>
-            <ANIMCURSOR_MIN>0</ANIMCURSOR_MIN>
-            <ANIMCURSOR_MAX>1</ANIMCURSOR_MAX>
-            <INVERT_ANIM>True</INVERT_ANIM>
-            <WWISE_EVENT>lswitch</WWISE_EVENT>
-            <NORMALIZED_TIME_1>0.1</NORMALIZED_TIME_1>
-            <NORMALIZED_TIME_2>0.5</NORMALIZED_TIME_2>
-        </UseTemplate>
+        <Component ID="#NODE_ID#" Node="#NODE_ID#">
+            <UseTemplate Name="ASOBO_GT_Switch_3States">
+                <INVERT_ANIM />
+                <WWISE_EVENT>lswitch</WWISE_EVENT>
+                <ANIMTIP_0>Set Console/Floor light to off</ANIMTIP_0>
```

Comment:
Can we try using the locPak file for this ?

We successfully did it for the loading tips, could be a place to start for localization